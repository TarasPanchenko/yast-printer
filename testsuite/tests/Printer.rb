# encoding: utf-8

module Yast
  class PrinterClient < Client
    def main
      # testedfiles: Printer.ycp

      Yast.include self, "testsuite.rb"
      TESTSUITE_INIT([], nil)

      Yast.import "Printer"

      DUMP("Printer::Modified")
      TEST(lambda { Printer.Modified }, [], nil)

      nil
    end
  end
end

Yast::PrinterClient.new.main
