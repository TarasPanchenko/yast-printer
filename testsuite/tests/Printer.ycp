{

// testedfiles: Printer.ycp

include "testsuite.ycp";
TESTSUITE_INIT([], nil);

import "Printer";

DUMP("Printer::Modified");
TEST(``(Printer::Modified()), [], nil);

}
