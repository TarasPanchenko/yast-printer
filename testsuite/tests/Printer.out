Dump	Printer::Modified
Return	false
